.travis.yml

```diff
@@ -2,6 +2,10 @@ sudo: required
 dist: trusty
 language: scala
 
+env:
+  - MVN_FLAG=
+  - MVN_FLAG=-Dminicluster-2.10.spark.version=2.0.1 -Dminicluster-2.11.spark.version=1.6.2
```

Comment:
Why test 2.10 with Spark2 and 2.11 with Spark 1.6, should we test official version?